DOC-3526: TinyMCE AI image edits now target the correct image (#4196)
tinymce/tinymce-docs: modules/ROOT/pages/8.7.0-release-notes.adoc

The following premium plugin updates were released alongside {productname} {release-version}.

=== TinyMCE AI

The {productname} {release-version} release includes an accompanying release of the **TinyMCE AI** premium plugin.

**TinyMCE AI** includes the following fix.

==== Image edits now target the correct image in content with multiple images
// #TINY-14454

> Previously, images sent to the model were not uniquely identified, so a prompt referencing a specific image could mistake one image for another and apply changes, such as alt text updates, to the wrong image. **TinyMCE AI** now tags each `+img+` element with a `+data-id+` attribute when content is sent to the model and removes the attribute before the response is inserted. Image edits now apply to the intended image, even when the content contains multiple images.

Reviewer: I think we shouldn't go so deep on the implementation details here just mention that we not tag them for the AI backend so they can be identified as unique images.

```suggestion
Previously, images sent to the model were not uniquely identified, so a prompt referencing a specific image could mistake one image for another and apply changes, such as alt text updates, to the wrong image. **TinyMCE AI** now uniquely tags each `+img+` elements so they are less likely to be mistaken. Image edits now apply to the intended image, even when the content contains multiple images.
```
